theano/tensor/tests/test_nlinalg.py

```diff
@@ -516,3 +518,29 @@ def test_numpy_compare(self):
             t_n = f(A[2][i])
             n_n = numpy.linalg.norm(A[2][i], A[3][i])
             assert _allclose(n_n, t_n)
+
+
+class TestLogAbsDet(unittest.TestCase):
+
+    def setUp(self):
+        utt.seed_rng()
+        self.op_class = LogAbsDet
+        self.op = logabsdet
+
+    def validate(self, input_mat):
+        x = theano.tensor.matrix()
+        f = theano.function([x], self.op(x))
+        out = f(input_mat)
+        svd_diag = numpy.linalg.svd(input_mat, compute_uv=False)
+        numpy_out = numpy.sum(numpy.log(numpy.abs(svd_diag)))
```

Comment:
I think my test cases are a bit silly, as the svd values are pretty big, about 1e+2 or so.. so maybe I should just rescale the test matrices (by e.g. 1 / sqrt(size)), then the error will probably be below the tolerance.